src/datta_lab_to_nwb/markowitz_gillis_nature_2023/depthvideointerface.py

```diff
@@ -0,0 +1,29 @@
+"""Primary class for converting raw depth video data."""
+from pynwb import NWBFile
+from .basevideointerface import BaseVideoInterface
+
+
+class DepthVideoInterface(BaseVideoInterface):
+    """Depth video interface for markowitz_gillis_nature_2023 conversion"""
+
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict) -> None:
+        video_metadata = dict(
+            Behavior=dict(
+                Videos=[
+                    dict(
+                        name="depth_video",
+                        description=(
+                            "Depth videos of mouse behaviour were acquired at 30Hz using a Kinect 2 for Windows "
+                            "(Microsoft) using a custom user interface written in Python (similar to ref. 60) on a Linux "
+                            "computer. For all OFA experiments, except where noted, mice were placed in a circular open "
+                            "field (US Plastics 14317) in the dark for 30min per experiment, for 2 experiments per day. "
+                            "As described previously, the open field was sanded and painted black with spray paint "
+                            "(Acryli-Quik Ultra Flat Black; 132496) to eliminate reflective artefacts in the depth video."
+                        ),
+                        unit="mm",
+                    )
+                ]
+            )
+        )
+        metadata.update(video_metadata)
```

Comment:
Why not have this be the `get_metdata` method for the interface?

Reply:
I don't think it will work to specify the videos in the `get_metadata` function bc they *both* get called *before* ``add_to_nwbfile`` and neuroconv's `VideoInterface` expects exactly the same number of video files as elements of `metadata['Behavior']['Videos']`.

I originally tried to include both in one videointerface, but I got an error saying that wasn't supported for `external_mode=False`.  

I think having 1 dataInterface : 1 videointerface : 1 video type (depth vs IR) is the cleanest implementation here since some sessions have both depth and IR but others have only depth.

Reply:
I see - this is a bug in the current design of that interface. It should have separate keys for each VideoInterface

Could you raise an issue about that on NeuroConv so we don't forget?